banned-characters rule prevents to use enableAllRules

**Describe the bug**
The documentation is misleading about the default configuration when using enableAllRules

**To Reproduce**
Steps to reproduce the behavior:
1. I updated revive `go get -u github.com/mgechev/revive`
2. I run it with the following flags & configuration file:

```shell
# no flags
revive file.go
```

```toml
# config file
ignoreGeneratedHeader = false
severity = "warning"
confidence = 0.8
errorCode = 0
warningCode = 0

# Enable all available rules
enableAllRules = true

# Disabled rules
[rule.blank-imports]
    Disabled = true
[rule.file-header]
    Disabled = true
[rule.max-public-structs]
    Disabled = true
[rule.line-length-limit]
    Disabled = true
[rule.function-length]
    Disabled = true

# Rule tuning
[rule.argument-limit]
    Arguments = [5]
[rule.cyclomatic]
    Arguments = [10]
[rule.cognitive-complexity]
    Arguments = [7]
[rule.function-result-limit]
    Arguments = [3]
[rule.error-strings]
    Arguments = ["mypackage.Error"]
```

So the exact block currently documented in the README.md file

**Expected behavior**
Have no panic, and all rules to be applied

**Logs**
panic: not enough arguments for banned-characters rule, expected 1, got 0. Please check the rule's documentation

**Desktop (please complete the following information):**
 - OS: [e.g. Ubuntu 22.04]
 - Version of Go 1.18.1

**Additional context**

The documentation needs to be updated to disable banned-characters rule when using enableAllRules

--- a/README.md
+++ b/README.md
@@ -341,6 +341,8 @@ enableAllRules = true
     Disabled = true
 [rule.function-length]
     Disabled = true
+[rule.banned-characters]
+    Disabled = true
 
 # Rule tuning
 [rule.argument-limit]

--- a/RULES_DESCRIPTIONS.md
+++ b/RULES_DESCRIPTIONS.md
@@ -123,7 +123,7 @@ _Configuration_: N/A
 
 _Description_: Checks given banned characters in identifiers(func, var, const). Comments are not checked.
 
-_Configuration_: This rule accepts a slice of strings, the characters to ban.
+_Configuration_: This rule requires a slice of strings, the characters to ban.
 
 Example:
 

